lighthouse-core/fraggle-rock/config/default-config.js

```diff
@@ -11,12 +11,14 @@ const {deepClone} = require('../../config/config-helpers.js');
 /** @type {LH.Config.AuditJson[]} */
 const frAudits = [
   'byte-efficiency/uses-responsive-images-snapshot',
+  'metrics/experimental-interaction-to-next-paint',
 ];
 
 /** @type {Record<string, LH.Config.AuditRefJson[]>} */
 const frCategoryAuditRefExtensions = {
   'performance': [
     {id: 'uses-responsive-images-snapshot', weight: 0},
+    {id: 'experimental-interaction-to-next-paint', weight: 30, group: 'metrics', acronym: 'INP'},
```

Comment:
Should we discuss audits to include in `relevantAudits`? Maybe as an item on https://github.com/GoogleChrome/lighthouse/issues/13916?

Reply:
> Should we discuss audits to include in `relevantAudits`? Maybe as an item on https://github.com/GoogleChrome/lighthouse/issues/13916?

added